node/src/Pos/Block/Worker.hs

```diff
@@ -97,11 +97,21 @@ blkOnNewSlot =
 -- Block creation worker
 ----------------------------------------------------------------------------
 
-blockCreator
-    :: WorkMode ssc ctx m
-    => SlotId -> SendActions m -> m ()
+blockCreator ::
+       forall ssc ctx m. WorkMode ssc ctx m
+    => SlotId
+    -> SendActions m
+    -> m ()
 blockCreator (slotId@SlotId {..}) sendActions = do
 
+    -- Print difference between tip slot and current slot.
+    -- TODO: probably not the best place, but it's not the biggest concern now.
+    tipHeader <- DB.getTipHeader @ssc
+    let tipSlot = epochOrSlotToSlot (getEpochOrSlot tipHeader)
+    let slotDiff = flattenSlotId slotId - flattenSlotId tipSlot
+    logInfo $ sformat ("Difference between current slot and tip slot is: "%int)
+              slotDiff
```

Comment:
Slightly better would be to name the whole thing and put it in the where clause, so then it doesn't clutter things up so much, and we know that `tipHeader` is not in scope and possibly used below. I suggest `logHowManySlotsBehind`.